style.css

```diff
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500;700&display=swap');
+
+* {
+  border: 0;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.menu-button {
+  display: flex;
+  justify-content: end;
+  margin-right: 10%;
+}
```

Comment:
Good job implementing the CSS Flexbox as it is part of the project requirements. 👍